src/FsAutoComplete.Core/Decompiler.fs

```diff
@@ -55,7 +55,7 @@ type TextWriterWithLocationFinder(tokenWriter:TextWriterTokenWriter, formattingP
     member __.MatchingLocation with get() = matchingLocation
 
 let decompilerForFile (file: string) =
-    let settings = DecompilerSettings(CSharp.LanguageVersion.Latest, ThrowOnAssemblyResolveErrors = true)
+    let settings = DecompilerSettings(CSharp.LanguageVersion.Latest, ThrowOnAssemblyResolveErrors = false)

```

Comment:
I should revert this. I ran into an error during this process but I need to turn this on so we have some logs for users.  Plus I think I solved the root issue anyway.